packages/terra-table/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "terra-table",
+  "main": "lib/index.js",
+  "version": "4.36.0",
+  "description": "The terra-table component provides user a way to render selectable data in a tabular format",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/table-framework.git",
+    "directory": "packages/terra-table"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-table",
+    "Table",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
```

Comment:
Should this be Oracle instead of Cerner?

Reply:
We haven't been given guidance on changing this but if this needs to be changed, we can probably go ahead do it and and update the rest of the packages in a separate PR.